listenbrainz/db/dump.py

```diff
@@ -278,7 +285,7 @@ def create_private_dump(location, dump_time, threads=DUMP_DEFAULT_THREAD_COUNT):
 
 def create_public_dump(location, dump_time, threads=DUMP_DEFAULT_THREAD_COUNT):
     """ Create postgres database dump for statistics and user info in db.
-        This includes a sanitized dump of the "user"' table and dumps of all tables
+        This includes a sanitized dump of the "user"' and recording_feedback table and dumps of all tables
```

Comment:
The recording_feedback table is included in "dumps of all tables". This addition makes the comment read poorly.

Reply:
Not sure I get you correctly. It reads _dumps of all tables in statistics schema_ or I misunderstood this?

Reply:
Do you really need to add "and recording_feedback" here?

Reply:
I don't feel strongly about it. I added it to specify that we also dump this table in the public dump. I can remove that if it makes the comment awkward.

Reply:
Let's do that. No need to highlight this table in particular.

Reply:
Okay

Reply:
Okay